WISE-Community/WISE
# Updated library project details to show correct content

To test:
- Load `/site` and click on projects in the homepage library. Make sure the project details dialog appears with the updated design (flat buttons, etc).
- Log in as a teacher and switch to the "Browse WISE Units" tab on the teacher homepage (`/site/teacher`). Click on projects from each of the three libraries. Make sure the project details dialog appears with the updated design. Make sure "Use with Class" appears as an option in the dialog.
- Switch to the "Class Schedule" tab and  open the run menu for some of your runs. Select "Unit Info" from the menu and make sure the project details dialog appears with the updated design. Make sure "Use with Class" does not appear as an option.

Closes #1476.
